AutoRunCustomTool64/source.extension.vsixmanifest

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="AutoRunCustomTool64.bdbd7370-6a02-43d2-8452-9e6e99d0533e" Version="1.0" Language="en-US" Publisher="Diligent Corporation" />
```

Comment:
Is "Diligent Corporation" the company you work for?
I appreciate the contribution, but it's still my extension :wink: